recipe/meta.yaml

```
@@ -34,6 +34,7 @@ requirements:
     - openssl {{ openssl }} # [linux]
   run:
     - python
+    - openssl 3.* # [linux]
```

```suggestion
    # exact pin handled through openssl run_exports
    - openssl # [linux]
```
We want to build both both OpenSSL 1 and 3.